insights/tests/datasources/test_system_user_dirs.py

```diff
@@ -0,0 +1,48 @@
+"""
+Tests for custom datasource for checking non-root owned packaged directories
+
+Author: Florian Festi <ffesti@redhat.com>
+"""
+import mock
+import rpm
```

Comment:
I met the same error  as reported in the CI, in my local test.
```
insights/tests/datasources/test_system_user_dirs.py:7: in <module>
    import rpm
E   ModuleNotFoundError: No module named 'rpm'
```
I think you might also need to make a `mock` `rpm.hdr()` in the following mocked `TransactionSet` class.

Reply:
My local tests passed because I created a new virtual environment like this: `python3.6 -m venv venv --system-site-packages`. It works even with python3.9.

Reply:
Okay, but with that in the test, I'm afraid that we can never get the CI being passed for the following PRs.

Reply:
Hi, I wrote to the code author, and we will try to improve the tests and remove the `rpm` module from it. Unfortunately, the CI will still fail because the `rpm` module is also used in the code.

Reply:
Hi @xiangce. I think we are fine with using the `rpm` module in tests. The `mock` module is used here only to mock testing data, and it is not necessary to mock the whole `rpm` module in tests. Do you agree with me?

Reply:
@jobselko the main concern is that we cannot rely on the CI tests due to the `rpm` dependency.  We'll have to test this manually.  The larger problem is that other developers won't be able to run pytest locally without tests failing or worse pytest failing to run due to the raised exceptions when imports fail.

Reply:
@bfahr I apologise for my confusion, but what do you suggest here as a solution?

Reply:
@jobselko - as mentioned above, it could be resolved by modifying the CI script:`.github/workflows/main.yml`. Please have a try according to your steps in local to all py26, py27, and py36.

Reply:
In a discussion on Slack, we agreed that a better solution is to use the `rpm` command instead of an `rpm` python package. Changes into the `.github/workflows/main.yml` file would be too complicated as the CI runs on an Ubuntu machine. I am closing this thread and will rewrite the code.